Cython/Compiler/ExprNodes.py

```diff
@@ -5934,6 +5934,17 @@ def generate_evaluation_code(self, code):
         if function.is_name or function.is_attribute:
             code.globalstate.use_entry_utility_code(function.entry)
 
+        abs_result = 'abs', 'labs', '__Pyx_abs_longlong'
+        is_signed_int = self.type.is_int and self.type.signed
+        if self.overflowcheck and is_signed_int and function.result() in abs_result:
+            code.globalstate.use_utility_code(UtilityCode.load_cached("Common", "Overflow.c"))
+            code.putln('if (unlikely(%s == __PYX_MIN(%s))) {\
+                PyErr_SetString(PyExc_OverflowError,\
+                                "Trying to take the   absolute value of the most negative integer is not defined"); %s; }' % (
```

Comment:
```suggestion
                                "There is one more negative than positive numbers in the range of signed integers. Consequently, the absolute value of the lowest possible negative integer leads to an overflow."); %s; }' % (
```

Reply:
There's a few extra spaces, but personally I prefer the shorter version. Most of the Python error messages for similar things are pretty concise.

Reply:
Sure thing. I just had to re-read the sentence a few times to figure it out, because it's IMO not a self-evident problem.

Reply:
@h-vetinari : I shamelessly borrowed that sentence from `man 3 abs` :-)